db/migrate/20160220123313_add_open_scap_result_entities.rb

```diff
@@ -0,0 +1,25 @@
+class AddOpenScapResultEntities < ActiveRecord::Migration
+  def up
+    create_table :openscap_results do |t|
+      t.belongs_to :container_image, :type => :bigint
+      t.bigint     :resource_id
+      t.string     :resource_type
+      t.datetime   :created_at
+    end
+
+    create_table :openscap_rule_results do |t|
+      t.belongs_to :openscap_result
```

Comment:
I don't remember if we have to specify type => bigint on association methods like belongs_to.  Either way, we specify it above and not here so let's make it consistent.